pkgs/development/python-modules/pyside/shiboken.nix

```diff
@@ -39,8 +42,17 @@ buildPythonPackage rec {
       \"$\{GENERATORRUNNER_PLUGIN_DIR}\" lib/generatorrunner/
   '';
 
-  # gcc6 patch was also sent upstream: https://github.com/pyside/Shiboken/pull/86
-  patches = [ ./gcc6.patch ] ++ (lib.optional (isPy35 || isPy36 || isPy37) ./shiboken_py35.patch);
+  patches = [
+    # gcc6 patch was also sent upstream: https://github.com/pyside/Shiboken/pull/86
+    ./gcc6.patch
+    (lib.optional (isPy35 || isPy36 || isPy37 || isPy38 || isPy39 ) ./shiboken_py35.patch)
+    (fetchurl {
+      # https://github.com/pyside/Shiboken/pull/90
```

Comment:
do you mind also commenting why this is relevant? easier when revisiting this

Reply:
`tp_print` was mostly used on python 2.x, unused on python 3.x and was [removed on python 3.9](https://docs.python.org/3/whatsnew/3.9.html#id3) - that's why pysideShiboken broke.

The patch just add a check to ignore `tp_print` on python 3.x.